contributing.md

```diff
@@ -0,0 +1,7 @@
+# Contributing
+
+## JS guidelines
+- Try not to use javascript if you don’t need it
+- If it really improves the experience, then use it
+- Include a fallback if JS isn’t supported (Good sanity check: if the fallback is just as good as the JS version, maybe you don’t need it!)
+- Test your JS with an integration test
```

Comment:
Is all of this still true for Drai(was taken from gcf)?

Reply:
Wow, I didn't even know that was in GCF. I have two thoughts:

- I think this advice is replaced by "Use Honeycrisp" (or maybe the advice should be moved to Honeycrisp)
- We won't be planning to maintain DRAI longterm, so we don't need a Contributing section.